### PR TITLE
refactor(egen): Refactored the Custom tabular bq managed dataset code as per template guidelines 

### PR DESCRIPTION
**Pull request by rohith-egen**
Refactored all the cells according to the template notebook
<br>

List of updates:
1. Steps are re-organized based on the [new template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb).
2. Renamed REGION as LOCATION
3. Removed import os - unused
4. Included --quite argument in the pip install dependencies.
5. Removed IS_TESTING env variable
6. Included colab enterprise link and generalized code for "colab only" cells
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 1]
```diff
@@ -32,23 +34,26 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fcustom%2Fcustom-tabular-bq-managed-dataset.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:44]: The URL in line 44 for the colab enterprise logo needs to be changed to:
- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 2]
```diff
@@ -128,181 +133,124 @@
...
-      "metadata": {
-        "id": "1fd00fa70a2a"
-      },
-      "outputs": [],
-      "source": [
-        "# Install the packages\n",
-        "! pip3 install --upgrade google-cloud-aiplatform \\\n",
-        "                        google-cloud-storage \\\n",
-        "                        'google-cloud-bigquery[pandas]'"
+        "## Getting Started"
```
Comment (kittyabs) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:139]: change "Getting started" to "Get started" to be in compliance with style guide. I've updated the template to reflect this change

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 3]
```diff
@@ -128,181 +133,124 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "blGlVGFYW9Pt"
+        "id": "dc848186ab0e"
       },
       "source": [
-        "### Colab only: Uncomment the following cell to restart the kernel."
+        "### Install Vertex AI SDK and other required packages"
```
Comment (kittyabs) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:148]: change to:
- Install Vertex AI SDK for Python and other required packages
    - to be in compliance with product guidelines

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 4]
```diff
@@ -128,181 +133,124 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "BaFKzJ_xXpvm"
+        "id": "54c5ef8a8f43"
       },
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated."
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:201]: Delete "Please" and contract "it is"

- Wait until it's finished...

File: notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 5]
```diff
@@ -4,10 +4,12 @@
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
+        "cellView": "form",
         "id": "copyright"
       },
       "outputs": [],
       "source": [
+        "# @title Copyright & License (click to expand)\n",
```
Comment (katiemn) [inline notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:12]: Should this be removed?

**Review by katiemn — APPROVED** [review]

**Pull request merged**